sympy/physics/control/lti.py

```diff
@@ -2343,6 +2385,31 @@ def _dummify_args(_arg, var):
 
     return dummy_arg_list, dummy_dict
 
+def _any_state_space_systems(arg_list: tuple) -> bool:
+    """
+    Check if there are any state space objects in the argument list.
+    If there are, return True, else return False.
+
+    Systems are considered to be state space systems if they are instances of
+    StateSpaceBase or have an attribute `is_StateSpace_object` that is True.
```

Comment:
Why is `is_StateSpace_object` needed? Why can't you simply use an isinstance check?